downloads/swan-lake-release-notes/swan-lake-preview6/RELEASE_NOTE.md

```diff
@@ -0,0 +1,354 @@
+---
+layout: ballerina-blank-page
+title: Release Note
+---
+### Overview of Ballerina Swan Lake Preview 6 
+
+This release is the sixth preview version of Ballerina Swan Lake. This release includes a new set of language features along with improvements and bug fixes to the compiler, runtime, standard libraries, and developer tooling.
```

Comment:
```suggestion
This release is the sixth preview version of Ballerina Swan Lake. This release includes a new set of language features along with improvements and bug fixes to the compiler, runtime, standard library, and developer tooling.
```

Reply:
Why this was resolved without committing the suggestion?

Reply:
I committed this but seems like something wrong has happened. Sorry, about it.